src/auth/GetWindowsCert.ts

```diff
@@ -0,0 +1,29 @@
+import * as ca from "win-ca";
+import * as caApi from "win-ca/api";
+import * as crypt from "crypto";
```

Comment:
Let's use the name of the imported module here:

```suggestion
import * as crypto from "crypto";
```

Reply:
Fixed